server.js

```diff
@@ -29,7 +29,11 @@ const handleError = (error, request, response, next) => { // eslint-disable-line
     try {
         let userRequest = request.query ? request.query.q : '';
         Helper.kodiShowError(request, response, userRequest + ': ' + publicError);
-    } catch {
+    }
+    
+    catch(err)
+    
+    {
```

Comment:
what kind of weired spacing style is that?
I fixed that myself now, but thank you!